packages/uniforms/src/SimpleSchemaBridge.js

```diff
@@ -202,8 +202,8 @@ export default class SimpleSchemaBridge extends Bridge {
             } else {
                 field = {
                     ...field,
-                    transform: value => options.find(option => option.value === value).label,
-                    allowedValues: options.map(option => option.value)
+                    transform: value => options.find(option => option.value.toString() === value).label,
+                    allowedValues: options.map(option => option.value.toString())
```

Comment:
But when you select 1 item, and the stuff gets re-rendered  the value is already a string, no? Let me check it tomorrow.